src/translations/recordTranslator.js

```diff
@@ -17,7 +20,7 @@ export class RecordTranslator {
    * @returns {Number} Returns visual row index.
    */
   toVisualRow(row) {
-    return this.hot.runHooks('unmodifyRow', row);
+    return this.hot.runHooks('unmodifyRow', this.getRowIndexMapper().getVisualIndex(row));
```

Comment:
We're killing hooks: `unmodifyRow`, `unmodifyCol`, `modifyRow` and `modifyCol`. It's unsafe to modify indexes outside of indexMapper.

Reply:
Excellent!

Reply:
A suggestion was abandoned. Case of doing sticky rows when sorting was adduced as an example that hook would be the easiest way of doing the feature.